geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/pubsub/SubCommandExecutor.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+
+package org.apache.geode.redis.internal.executor.pubsub;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.geode.redis.internal.executor.Executor;
+import org.apache.geode.redis.internal.executor.RedisResponse;
+import org.apache.geode.redis.internal.netty.Command;
+import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
+
+public class SubCommandExecutor implements Executor {
+
+  static final String UNKNOWN_SUBCOMMAND_ERROR_BASE =
+      "Unknown subcommand or wrong number of arguments for '%s'";
```

Comment:
I'm guessing we'll add the `Try PUBSUB HELP.` message to the end of this once `PUBSUB HELP` is implemented?  I think we should put this message in the `RedisConstants` file where other similar error messages live and rename it to follow the same pattern as the others, maybe `ERROR_UNKNOWN_PUBSUB_SUBCOMMAND` since it will eventually be specific to PUBSUB.... we could actually refactor those subcommand error messages to all use the same base and insert the relevant primary command, so something like:
```
ERROR_UNKNOWN_SUBCOMMAND =
      "Unknown subcommand or wrong number of arguments for '%1$s'. Try %2$s HELP."

Reply:
thanks-  for some reason I didn't find that file when I looked for it-  thought maybe it had been removed.  

as far as the "help" bit, yeah, when/if we implement help I think we should add that bit back to message.  It's not in the group of stories currently pointed.  I think leaving it off the error message until we implement help makes the most sense?    at that point, maybe it would make sense to genericize all the subcommands error messages....